_posts/izayak/2016-02-03-ch5-exercise.md

```diff
@@ -0,0 +1,12 @@
+---
+author: izayak
+title: "Ruotong's Chapter 5 Loops Exercise"
+layout: post
+---
+
+<iframe src="https://trinket.io/embed/python/cb8bf3c733" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
Mostly good.  When I entered 5, 10, 0 in exercise 2, though, the average was shown as 10, not 5.  Can you figure out why?